setup/environment-linux.yml

```diff
@@ -1,148 +1,157 @@
     - 0x-contract-wrappers==2.0.0
     - 0x-json-schemas==2.1.0
     - 0x-order-utils==4.0.0
-    - aioconsole==0.1.13
-    - aiohttp==3.6.1
-    - aiokafka==0.5.0
+    - aioconsole==0.1.15
+    - aiohttp==3.6.2
+    - aiokafka==0.5.2
     - appdirs==1.4.3
-    - aspy-yaml==1.3.0
+    - asn1crypto==1.3.0
     - async-timeout==3.0.1
     - attrdict==2.0.1
-    - attrs==18.2.0
-    - autobahn==19.2.1
-    - automat==0.7.0
-    - cachetools==3.0.0
-    - cfgv==2.0.1
+    - attrs==19.3.0
+    - autobahn==20.3.1
+    - automat==20.2.0
+    - base58==2.0.0
+    - cachetools==4.0.0
+    - cffi==1.14.0
+    - cfgv==3.1.0
     - chardet==3.0.4
-    - coincurve==12.0.0
+    - coincurve==13.0.0
     - constantly==15.1.0
-    - cytoolz==0.9.0.1
-    - dateparser==0.7.1
+    - cryptography==2.8
+    - cytoolz==0.10.1
+    - dateparser==0.7.4
+    - decorator==4.4.2
+    - deprecated==1.2.7
+    - distlib==0.3.0
     - entrypoints==0.3
-    - eth-abi==2.1.0
-    - eth-account==0.3.0
+    - eth-abi==2.1.1
+    - eth-account==0.4.0
     - eth-bloom==1.0.3
     - eth-hash==0.2.0
     - eth-keyfile==0.5.1
-    - eth-keys==0.2.1
+    - eth-keys==0.2.4
     - eth-rlp==0.1.2
-    - eth-typing==2.1.0
-    - eth-utils==1.4.1
-    - flake8==3.7.8
-    - future==0.17.1
-    - hexbytes==0.1.0
-    - https://hummingbot-python.s3-us-west-2.amazonaws.com/hummingsim-20191212.3-cp36-cp36m-linux_x86_64.whl
-    - hyperlink==18.0.0
-    - hypothesis==4.23.8
-    - identify==1.4.7
-    - idna-ssl==1.1.0
-    - importlib-metadata==0.23
-    - importlib-resources==1.0.2
+    - eth-typing==2.2.1
+    - eth-utils==1.8.4
+    - filelock==3.0.12
+    - flake8==3.7.9
+    - future==0.18.2
+    - hexbytes==0.2.0
+    - hyperlink==19.0.0
+    - identify==1.4.11
+    - idna==2.9
     - incremental==17.5.0
-    - jsonschema==3.0.1
-    - kafka-python==1.4.3
+    - ipfshttpclient==0.4.12
+    - jsonschema==3.2.0
+    - kafka-python==1.4.6
     - lru-dict==1.1.6
     - mccabe==0.6.1
-    - more-itertools==7.2.0
-    - multidict==4.5.2
-    - mypy-extensions==0.4.1
-    - nodeenv==1.3.3
+    - multiaddr==0.0.9
+    - multidict==4.7.5
+    - mypy-extensions==0.4.3
+    - netaddr==0.7.19
+    - nodeenv==1.3.5
+    - objgraph==3.4.1
     - parsimonious==0.8.1
-    - pre-commit==1.18.2
-    - py-ecc==1.7.0
-    - pyasn1==0.4.5
-    - pyasn1-modules==0.2.4
+    - pre-commit==2.1.1
+    - protobuf==3.11.3
+    - pyasn1==0.4.8
+    - pyasn1-modules==0.2.8
     - pycodestyle==2.5.0
-    - pycryptodome==3.7.3
+    - pycparser==2.20
+    - pycryptodome==3.9.7
     - pyflakes==2.1.1
-    - pyhamcrest==1.9.0
-    - PyJWT==1.7.1
+    - pyhamcrest==2.0.2
+    - pyinstall==0.1.4
+    - pyjwt==1.7.1
+    - pyopenssl==19.1.0
     - pyperclip==1.7.0
-    - pyrsistent==0.15.2
+    - pyrsistent==0.15.7
     - python-binance==0.7.1
-    - python-telegram-bot==11.1.0
-    - regex==2019.2.21
-    - requests==2.21.0
-    - rlp==1.1.0
-    - ruamel-yaml==0.15.89
+    - python-graphviz==0.13.2
```

Comment:
@martinkou 
I think this should be: `graphviz==0.13.2` instead of `python-graphviz==0.13.2`


![image](https://user-images.githubusercontent.com/12274903/76171382-e1f93780-6147-11ea-9e98-0fbe31074779.png)

Reply:
That's what's causing the jenkins build to fail